### PR TITLE
Create Chrisbase12.json

### PR DESCRIPTION
**Pull request by OverCharred**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by Stef-DP — APPROVED** [review]

File: domains/_vercel.chrisbase12.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "username": "insert-github-username-here"
```
Comment (Stef-00012) [inline domains/_vercel.chrisbase12.json:3]: this should be your github username

**Review by Stef-DP — DISMISSED** [review]

File: domains/_vercel.chrisbase12.json (hunk 1 above)
Comment (CuteDog5695) [inline domains/_vercel.chrisbase12.json:3]:
```suggestion
    "username": "OverCharred"
```

File: domains/chrisbase12.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "username": "overcharred"
```
Comment (CuteDog5695) [inline domains/chrisbase12.json:3]:
```suggestion
    "username": "OverCharred"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**